### DISCUSSION TITLE
Implementation questions

### DISCUSSION THREAD

**Discussion by skylarr1227** [InterStella0/starlight-dpy#3] [Q&A] [answered]
My apologies, and yeah I understand why you wouldnt give "contact" information however I didnt assume discord username generally would fall into the "dont give out" category but i get it-I was actually interested in what you've made here and just wanted to see if you would be interested possibly in working with our project... seemed like the wrong place to ask on github tho.. meh.

Well-okay so based on what i had previously stated though-and its a bit difficult to explain and show what i mean while on this platform but our bots commands are all hybrid or pure app_command slash... you said you would suggest against this but my question then is more towards why? Currently our help menu is purely static text describing commands (project is the result of almost 4.5 years of work .. help cog hasnt been touched in ages) seems like it would have to be better than that or the built in help with D.py but perhaps im wrong.

**Comment by InterStella0** [comment, marked as answer]
I don't have any interest in working on people's projects as I don't have time to be involved in it and I don't do things for free.

I'm also confused about what do you mean by me being against app_command slash?

**Reply by skylarr1227** [reply]
I never said whether it was free or not, as it generally wouldnt be-and dw about it that answers my question. Ill figure it out. I never said you were against app_command... i said you said you;d suggest against using any of this on a large bot. o_O and was wondering why.

**Reply by InterStella0** [reply]
I see, my reasoning for being against using this library in production is that I'm not yet satisfied with the current implementation for most of the features. I'm doing several testing for a lot of the features to have a solid grasp on how people may use them. This library is still in its drafting stage. I may remove a feature if I don't like it.

There are also external reasons that may cause the removal of a feature. Exhibit A, I may remove the `SimplePaginationView` class. I've heard that Danny was thinking of a simple implementation for pagination similar to `discord-ext-menus` but for View. Once this is rolled out, I would completely remove it from this library and utilize that feature instead for better compatibility with the current 'tech'.

But if you insist on using this library, I wouldn't stop you. However, you would need to re-implement things in case there are breaking changes.

**Reply by skylarr1227** [reply]
And i would love to help with testing these features in a working environment and be able to provide you with actual feedback if that would help from my users use of it. 

I understand fully what comes with that and ANY amount of it working atm i feel would be beneficial :P 

Also-yes, i understand and am aware of changes :) generally with someone whos clearly as skilled as you are changes would be for the better, and if they were not i would not change from the point when they were providing me with benefit :P I certainly dont expect anything else. <3 Keep in mind i think this so far is AWESOME-and from my test bot-it works freaking great at the moment-gonna swap to live and see-but yeah, any future updates i only see being even better. I am sorry if i communicated wrong here or gave the wrong impression that was not my intention.
